packages/turf-center-of-mass/README.md

```diff
@@ -9,7 +9,9 @@ using this formula: [Centroid of Polygon](https://en.wikipedia.org/wiki/Centroid
 
 **Parameters**
 
--   `feature` **[Feature](http://geojson.org/geojson-spec.html#feature-objects)** the feature
+-   `feature` **[Feature](http://geojson.org/geojson-spec.html#feature-objects)** or
```

Comment:
these docs are generated from source code and will be blown away again when [the readme generation script runs](https://github.com/Turfjs/turf/blob/master/scripts/generate-readmes)